PKGBUILD

```diff
@@ -1,17 +1,15 @@
-# Maintainer: rHermes <teodor_spaeren@riseup.net>
-# Maintainer: bnavigator <code@bnavigator.de>
-# Contributor: PlusMinus
-# Contributor: rhabbachi
+# Maintainer: eric-dev <eric2043@gmail.com>
```

Comment:
Should I add the previous maintainers/contributors as well?

Reply:
Yes

Reply:
I think the confusing aspect, is that I'm not intending to take ownership of https://aur.archlinux.org/packages/displaylink

but creating a new package `displaylink-runit`

Reply:
> I think the confusing aspect, is that I'm not intending to take ownership of https://aur.archlinux.org/packages/displaylink
> 
> but creating a new package `displaylink-runit`

Yes, but the old maintainers have still contributed to your new package.
Thats just out of respect. You are taking their old code, modify a bit and their "old" code is still present. So respect it and keep them as contributors.
You can also find this in the packaging rules in the archlinux wiki.

Reply:
Awesome, thanks for confirminig with me!

Reply:
@ptr1337 Do you wanted to be added as a contributor? Since you reviewed my code :smile: